src/setup/platformSetup/index.website.ts

```diff
@@ -21,23 +21,25 @@ function webUpdate() {
 
             if (!Visibility.isVisible()) {
                 // Page is hidden, refresh immediately
-                window.location.reload(true);
+                window.location.reload();
                 return;
             }
 
             // Prompt user to refresh the page
             if (window.confirm('Refresh the page to get the latest updates!')) {
-                window.location.reload(true);
+                window.location.reload();
             }
         });
 }
 
+type WebUpdater = () => {
+    init: () => void;
+    update: () => void;
+};
```

Comment:
Not a blocker but could we not reuse `PlatformSpecificUpdater` insted here, somehow? 

https://github.com/Expensify/App/blob/b8659329560a13d2d479c7920c9a73200d785d24/src/libs/checkForUpdates.ts#L3-L6

Reply:
I feel like it would be a little cleaner that way, in that we wouldn't have this duplicate type definition. But in a way, it also feels a little over-complicated for just this simple type... 

In the end I don't have a strong opinion, @parasharrajat I'm curious what you'd recommend here.